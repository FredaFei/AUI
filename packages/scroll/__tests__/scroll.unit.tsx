import * as renderer from 'react-test-renderer'
import React from 'react'
import { mount} from 'enzyme'

import Scroll from '../scroll'

describe('测试 Scroll 以下行为', () => {
  it('Scroll 存在', () => {
    const json = renderer.create(<Scroll/>).toJSON()
    expect(json).toMatchSnapshot()
  })
  it('Scroll 设置 pullingHeight', () => {
    const wrapper = mount(<Scroll pullingHeight={200}>apple</Scroll>)
    expect(wrapper.props().pullingHeight).toEqual(200)
  })

})