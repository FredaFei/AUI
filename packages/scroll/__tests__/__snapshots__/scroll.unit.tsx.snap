// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`测试 Scroll 以下行为 Scroll 存在 1`] = `
<div
  className="am-scroll-wrapper "
>
  <div
    className="am-scroll-inner"
    onScroll={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    style={
      Object {
        "right": -0,
        "transform": "translateY(0px)",
      }
    }
  />
  <div
    className="am-scroll-track"
  />
</div>
`;
